src/foam/i18n/TranslationService.js

```diff
@@ -18,9 +18,26 @@ foam.INTERFACE({
     {
       name: 'getTranslation',
       async: true,
-      args: [ 'String locale', 'String source' ],
-//      javaType: 'foam.i18n.Locale'
-type: 'String'
+      args: [
+        {
```

Comment:
Why do we need 'x'?

Reply:
@kgrgreer I wanted to call x.get("localeDAO") in getTranslation method.

Reply:
Call getX() instead.